packages/core/src/lib/map/map.ts

```diff
@@ -187,23 +206,20 @@ export function mapMutate<
  * @param {Mapper} mapper - the mapper instance
  * @param {ErrorHandler} errorHandler - the error handler
  * @param {Function} setMemberFn
+ * @param {Function} getMemberFn
  * @param {boolean} [isMapArray = false] - whether the map operation is in Array mode
  */
-function map<
-  TSource extends Dictionary<TSource> = any,
-  TDestination extends Dictionary<TDestination> = any
->(
-  sourceObj: TSource,
-  mapping: Mapping<TSource, TDestination>,
-  options: MapOptions<TSource, TDestination>,
-  mapper: Mapper,
-  errorHandler: ErrorHandler,
-  setMemberFn: (
-    destinationMemberPath: string[],
-    destination?: TDestination
-  ) => (value: unknown) => void,
-  isMapArray = false
-) {
+function map<TSource extends Dictionary<TSource> = any,
+  TDestination extends Dictionary<TDestination> = any>({
+  sourceObj,
+  mapping,
+  options,
+  mapper,
+  errorHandler,
+  setMemberFn,
+  getMemberFn,
+  isMapArray
```

Comment:
nitpick: I know this is similar but please default `isMapArray` to false?

Reply:
Updated via fixup and rebase.